modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/lid-driven-two-phase.i

```diff
@@ -0,0 +1,364 @@
+    x = vel_x
+    y = vel_y
+  []
+  [populate_u_slip]
+    type = WCNSFV2PSlipVelocityAux
+    variable = 'vel_slip_x'
+    momentum_component = 'x'
+    u = 'vel_x'
+    v = 'vel_y'
+    rho = ${rho}
+    mu = 'mu_mixture'
+    rho_d = ${rho_d}
+    particle_diameter = ${dp}
+    linear_coef_name = 'Darcy_coefficient'
+  []
+  [populate_v_slip]
+    type = WCNSFV2PSlipVelocityAux
+    variable = 'vel_slip_y'
+    momentum_component = 'y'
+    u = 'vel_x'
+    v = 'vel_y'
+    rho = ${rho}
+    mu = 'mu_mixture'
+    rho_d = ${rho_d}
+    particle_diameter = ${dp}
+    linear_coef_name = 'Darcy_coefficient'
+  []
+  [populate_cd]
+    type = FunctorAux
+    variable = drag_coefficient
+    functor = 'Darcy_coefficient'
+  []
+  [populate_rho_mixture_var]
+    type = FunctorAux
+    variable = rho_mixture_var
+    functor = 'rho_mixture'
+  []
+  [populate_mu_mixture_var]
+    type = FunctorAux
+    variable = mu_mixture_var
+    functor = 'mu_mixture'
+  []
+  [compute_phase_1]
+    type = ParsedAux
+    variable = phase_1
+    coupled_variables = 'phase_2'
+    expression = '1 - phase_2'
+  []
+[]
+
+[Materials]
+  [CD]
+    type = NSFVDispersePhaseDragMaterial
+    rho = 'rho_mixture'
+    mu = mu_mixture
+    u = 'vel_x'
+    v = 'vel_y'
+    particle_diameter = ${dp}
+  []
+  [mixing_material]
+    type = NSFVMixtureMaterial
+    phase_1_names = '${rho_d} ${mu_d}'
+    phase_2_names = '${rho} ${mu}'
+    prop_names = 'rho_mixture mu_mixture'
+    phase_1_fraction = 'phase_2'
+  []
+[]
+
+[Postprocessors]
+  [average_void]
+    type = ElementAverageValue
+    variable = 'phase_2'
+  []
+  [max_y_velocity]
+    type = ElementExtremeValue
+    variable = 'vel_y'
+    value_type = max
+  []
+  [min_y_velocity]
+    type = ElementExtremeValue
+    variable = 'vel_y'
+    value_type = min
+  []
+  [max_x_velocity]
+    type = ElementExtremeValue
+    variable = 'vel_x'
+    value_type = max
+  []
+  [min_x_velocity]
+    type = ElementExtremeValue
+    variable = 'vel_x'
+    value_type = min
+  []
+  [max_x_slip_velocity]
+    type = ElementExtremeValue
+    variable = 'vel_slip_x'
+    value_type = max
+  []
+  [max_y_slip_velocity]
+    type = ElementExtremeValue
+    variable = 'vel_slip_y'
+    value_type = max
+  []
+  [max_drag_coefficient]
+    type = ElementExtremeFunctorValue
+    functor = 'drag_coefficient'
+    value_type = max
+  []
+[]
+
+[Preconditioning]
+  [smp]
+    type = SMP
+    full = true
+  []
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'NEWTON'
+  petsc_options_iname = '-pc_type -pc_factor_shift_type'
+  petsc_options_value = 'lu NONZERO'
+  [TimeStepper]
+    type = IterationAdaptiveDT
+    optimal_iterations = 7
+    iteration_window = 2
+    growth_factor = 2.0
+    cutback_factor = 0.5
+    dt = 1e-3
+  []
+  nl_max_its = 10
+  steady_state_detection = true
+  steady_state_tolerance = 1e-10
+  nl_rel_tol = 1e-03
+  nl_abs_tol = 1e-11
+  l_max_its = 5
+  residual_and_jacobian_together = true
```

Comment:
I think this is the default for FV now, @lindsayad ?

Reply:
Yes. Removed line